velox/vector/fuzzer/VectorFuzzer.h

```diff
@@ -285,6 +291,10 @@ class VectorFuzzer {
       const std::vector<TypePtr>& scalarTypes,
       int maxDepth = 5);
 
+  size_t randInRange(size_t min, size_t max);
+
+  TypePtr randMapType(int maxDepth = 5);
```

Comment:
could you document what exactly this method does? (e.g., can both keys and values be nested?)